pkg/formatted/color.go

```diff
@@ -58,7 +58,7 @@ func DecorateAttr(attrString, message string) string {
 		return "📨 "
 	case "outputresources":
 		return "📡 "
-	case "steps":
+	case "steps", "check":
```

Comment:
Why a foot for check?

Reply:
I just thought to use the same as step, as check is a step basically but open to use anything else

Reply:
This is why I think we need to seriously discuss getting rid of emojis 😢. While they are conceptually similar, the foot is there cause it's supposed to represent a step. A check would maybe be more aligned with ✔️ , but maybe it's just easier to not decorate it.

Reply:
Yes added :heavy_check_mark: emoji for check